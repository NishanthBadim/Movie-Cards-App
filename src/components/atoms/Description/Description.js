import React from 'react';
import PropTypes from 'prop-types'


/**
- Use this component for providing description.
**/

const Description =(props) => {
    return(
        <p className="text-justify" style={{fontSize: '14px'}}>
            {props.description}
        </p>
    );
}

Description.propTypes= {
    /**
    Use the loading state to indicate that the data Avatar needs is still loading.
    */
    description: PropTypes.string,
}
export default Description;