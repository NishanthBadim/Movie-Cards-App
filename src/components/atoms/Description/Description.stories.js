import React from 'react'
import {withKnobs, text } from '@storybook/addon-knobs/react'
import Description from './Description'
import { ColorPalette } from '@storybook/components'

export default {
    component: Description,
    title: "Description",
    decorators: [withKnobs],
    parameters: {
        componentSubtitle: 'Gives the description of a movie'
    }
}

export const Default = () => {
    return (
        <Description description={text("text","Here goes the description of movie")} />
    )
}