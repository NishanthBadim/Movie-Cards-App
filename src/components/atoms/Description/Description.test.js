import React from 'react';
import {create} from 'react-test-renderer'
import "@testing-library/jest-dom/extend-expect";
import Description from './Description'

test("Match the snapshot - Description",() => {
    const input=create(<Description/>);
    expect(input.toJSON()).toMatchSnapshot();
})