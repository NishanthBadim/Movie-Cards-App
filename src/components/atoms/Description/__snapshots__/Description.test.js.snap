// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match the snapshot - Description 1`] = `
<p
  className="text-justify"
  style={
    Object {
      "fontSize": "14px",
    }
  }
/>
`;
